src/orange/OrangeTypes.hh

```diff
@@ -115,6 +115,17 @@ enum class SurfaceType : unsigned char
     size_  //!< Sentinel value for number of surface types
 };
 
+//---------------------------------------------------------------------------//
+/*!
+ * Enumeration for mapping transform implementations to integers.
+ */
+enum class TransformType : unsigned char
+{
+    tl,  //!< Translation only
```

Comment:
The other enums in orange use full words, e.g. `BoundaryResult::reentrant` and `BoundaryResults::exiting` instead of `BoundaryResult::re` and `BoundaryResults::ex`, which I think is preferable for readability. How about `TransformType::translation` and `TransformType::transformation`?

Reply:
The other enums except for the "surface types" which this new type imitates: but admittedly those do have a Fortran heritage. Maybe we should plan to change the existing surface type enums to class-imitating names? (I'm ok with that.)

Reply:
I think the fact that the SurfaceTypes have a heritage (whereas `tl`/`tf` are more ad hoc) makes them more acceptable. I think `plane_x` is better than `px`, but I also don't think `px` is hurting anything.